paymentsheet/src/main/java/com/stripe/android/link/LinkActivityViewModel.kt

```diff
@@ -285,7 +296,7 @@ internal class LinkActivityViewModel @Inject constructor(
 
                 DaggerNativeLinkComponent
                     .builder()
-                    .configuration(args.configuration)
+                    .configuration(args.configurationWithUpdatedCustomerInfo)
```

Comment:
Might be weird that this happens here, but if we don't update the `configuration`, then [this code](https://github.com/stripe/stripe-android/blob/master/paymentsheet/src/main/java/com/stripe/android/link/account/DefaultLinkAccountManager.kt#L399) will fetch the Link account associated with that email address.